src/components/app/details/triggerView/workflow/Workflow.tsx

```diff
@@ -15,7 +16,9 @@ export class Workflow extends Component<WorkflowProps> {
                 return this.renderSourceNode(node)
             } else if (node.type === 'CI') {
                 return this.renderCINodes(node)
-            } else if (node.type === 'PRECD' || node.type === 'POSTCD') {
+            } else if (node.type === PipelineType.WEBHOOK) {
+              return this.renderWebhookNode(node)
+          } else if (node.type === 'PRECD' || node.type === 'POSTCD') {
```

Comment:
Also, include PRECD & POSTCD to the same const/enum type.

Reply:
done